hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/NettyRpcServer.java

```diff
@@ -67,20 +68,49 @@
    * Tests may set this down from unlimited.
    */
   public static final String HBASE_NETTY_EVENTLOOP_RPCSERVER_THREADCOUNT_KEY =
-    "hbase.netty.eventloop.rpcserver.thread.count";
+      "hbase.netty.eventloop.rpcserver.thread.count";
   private static final int EVENTLOOP_THREADCOUNT_DEFAULT = 0;
 
+  protected static final String SERVER_TCP_BACKLOG = "hbase.ipc.server.tcpbacklog";
+  protected static final String SERVER_TCP_REUSEADDR = "hbase.ipc.server.tcpreuseaddr";
+  protected static final String SERVER_TCP_NODELAY = "hbase.ipc.server.tcpnodelay";
```

Comment:
`NODELAY` and `KEEPALIVE` are unused. Also managed by the parent class.

Reply:
Same copy paste mistake as above - previous PR had this all in the base class.